staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -439,6 +439,15 @@ const (
 	//
 	// The annotation is added to a "Bookmark" event.
 	InitialEventsAnnotationKey = "k8s.io/initial-events-end"
+
+	// InitialEventsListBlueprintAnnotationKey is the name of the key
+	// where an empty, base64-encoded, versioned list is stored as a string.
+	//
+	// Used by clients that want to reconstruct a list object from
+	// the retrieved stream, ensuring it is equivalent to a standard LIST call.
+	//
+	// The annotation is added to a "Bookmark" event.
+	InitialEventsListBlueprintAnnotationKey = "kubernetes.io/initial-events-list-blueprint"
```

Comment:
I'm conceptually fine with the approach, but how exactly the annotation should be named and whether the comment is convincing enough, I would leave to @deads2k

Reply:
ref for those wondering wy kubernetes.io: https://github.com/kubernetes/community/blob/e00e0ada0c01d950debd196af7e96cc3218adf70/contributors/devel/sig-architecture/api-conventions.md?plain=1#L1858-L1859

> - "kubernetes.io" is the preferred form for labels and annotations, "k8s.io" should not be used
      for new map keys.

past mistakes do not justify future mistakes.